specs/latest/2.0/index.html

```diff
@@ -1869,7 +1869,7 @@ <h2 class="no-toc">Table of contents</h2>
         </dt>
         <dd>
           <p>If no WebGLBuffer is bound to the <code>PIXEL_PACK_BUFFER</code> target, generates an <code>INVALID_OPERATION</code> error.</p>
-          <p><em>offset</em> is the byte offset into the WebGLBuffer's data store; generates an <code>INVALID_VALUE</code> error if it's less than 0.</p>
+          <p><em>offset</em> is the byte offset into the WebGLBuffer's data store; generates an <code>INVALID_VALUE</code> error if it's less than 0. If <em>offset</em> passes the end of the pixel pack buffer, or the pixel pack buffer is not large enough to store all pixels, generates an <code>INVALID_OPERATION</code>.</p>
```

Comment:
If the offset passes the end of the buffer, which error does the browser should generate, INVALID_ENUM or INVALID_OPERATION? 

bufferSubData generates INVALID_ENUM in this situation, but other APIs, say vertexAttribPointer, would generate INVALID_OPERATION. 

Personally, I prefer to generate INVALID_ENUM, WDYT?